src/services/fileStorage/hooks/model-hooks.js

```diff
@@ -23,6 +24,11 @@ const restrictToCurrentUser = (hook) => {
 	});
 };
 
+const updateParents = async (hook) => {
+	await updateParentDirectories(hook.id);
```

Comment:
Just a question for my own, better understanding: Would it be more efficient to not use await here?

Reply:
And if removing await could cause problems, what kind of problems would that be?

Reply:
I would probably work and the response would be faster. My thoughts were mainly based on reliability: When not using async/await, the request doesn't care about errors in the method and the sender can't be sure wether the parent directories updated successfully. The question is: Do we really need that? Probably it's not that important and therefore I could just call the async function and return the context immediately.

Reply:
Just tested it without `async/await` and it seems to work the same. Don't know what's better practice in this case...

Reply:
Now it runs asynchronously and logs possible errors:
```JavaScript
const updateParents = (hook) => {
	updateParentDirectories(hook.id).catch((error) => {
		asyncErrorLog(error, 'Failed to update parent directories.');
	});
	return hook;
};
```
Is this the error handler you meant in your reply @Parnswir ?

Reply:
I think this would be it, but @CeEv can probably tell you more